mkdocs_awesome_pages_plugin/tests/e2e/test_gen_files.py

```diff
@@ -48,7 +49,7 @@ def write_files(path):
 class TestGeneratedFiles(E2ETestCase):
     """See https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin/issues/78."""
 
-    PLUGINS = [GeneratedFiles]
+    PLUGINS = (GeneratedFiles,)
```

Comment:
I'm just curious, why the change from list to tuple?

Reply:
So that the parent class default is immutable (same thing with function argument defaults)

Reply:
Gotcha